### PR TITLE
Translate `worldedit_commands` commands

### PR DESCRIPTION
**Pull request by alek13**
just add translations for commands descriptions in `worldedit_commands`

### CODE REVIEW

File: worldedit_commands/init.lua [hunk 1]
```diff
@@ -419,68 +431,68 @@ worldedit.register_command("unmark", {
...
 			worldedit.set_pos[name] = "pos1only"
-			worldedit.player_notify(name, "select position 1 by punching a node")
+			worldedit.player_notify(name, S("select position @1 by punching a node", 1))
 		elseif param == "set2" then --set WorldEdit position 2
 			worldedit.set_pos[name] = "pos2"
-			worldedit.player_notify(name, "select position 2 by punching a node")
+			worldedit.player_notify(name, S("select position @1 by punching a node", 2))
 		elseif param == "get" then --display current WorldEdit positions
 			if worldedit.pos1[name] ~= nil then
 				worldedit.player_notify(name, "position 1: " .. minetest.pos_to_string(worldedit.pos1[name]))
```
Comment (sfan5) [inline worldedit_commands/init.lua:480]: missing translation

File: worldedit_commands/locale/worldedit.ru.tr [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# textdomain: worldedit
```
Comment (sfan5) [inline worldedit_commands/locale/worldedit.ru.tr:1]: can you add a template file too?

**Comment by sfan5** [comment]
Works fine but you missed a bunch:
```
worldedit_commands/safe.lua:	worldedit.player_notify(name, "WARNING: this operation could affect up to " .. count .. " nodes; type //y to continue or //n to cancel")
worldedit_commands/safe.lua:			worldedit.player_notify(name, "no operation pending")
worldedit_commands/safe.lua:			worldedit.player_notify(name, "no operation pending")
worldedit_commands/init.lua:		worldedit.player_notify(name, "Schematic was created with a newer version of WorldEdit.")
worldedit_commands/init.lua:		worldedit.player_notify(name, "region reset")
worldedit_commands/init.lua:		worldedit.player_notify(name, "region marked")
worldedit_commands/init.lua:		worldedit.player_notify(name, "region unmarked")
worldedit_commands/init.lua:		worldedit.player_notify(name, "current region has a volume of " .. volume .. " nodes ("
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes replaced")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes replaced")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes copied")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes moved")
worldedit_commands/init.lua:			worldedit.player_notify(name, count .. " nodes stacked")
worldedit_commands/init.lua:			worldedit.player_notify(name, count .. " nodes stacked")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes stretched")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes transposed")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes flipped")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes rotated")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes oriented")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes updated")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes updated")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes removed")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes hidden")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes suppressed")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes highlighted")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes restored")
worldedit_commands/init.lua:			worldedit.player_notify(name, "Could not save file to \"" .. filename .. "\"")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes saved")
worldedit_commands/init.lua:			worldedit.player_notify(name, "Schematic empty, nothing allocated")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes allocated")
worldedit_commands/init.lua:			worldedit.player_notify(name, "Loading failed!")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " nodes loaded")
worldedit_commands/init.lua:			worldedit.player_notify(name, "code error: " .. err)
worldedit_commands/init.lua:			worldedit.player_notify(name, "code successfully executed", false)
worldedit_commands/init.lua:			worldedit.player_notify(name, "code error: " .. err, false)
worldedit_commands/init.lua:			worldedit.player_notify(name, "code successfully executed", false)
worldedit_commands/init.lua:			worldedit.player_notify(name, "Failed to create Minetest schematic")
worldedit_commands/init.lua:			worldedit.player_notify(name, "Saved Minetest schematic to " .. param)
worldedit_commands/init.lua:			worldedit.player_notify(name, "failed to place Minetest schematic")
worldedit_commands/init.lua:			worldedit.player_notify(name, "placed Minetest schematic " .. param ..
worldedit_commands/init.lua:			worldedit.player_notify(name, "select Minetest schematic probability values by punching nodes")
worldedit_commands/init.lua:			worldedit.player_notify(name, "finished Minetest schematic probability selection")
worldedit_commands/init.lua:			worldedit.player_notify(name, "currently set node probabilities:")
worldedit_commands/init.lua:			worldedit.player_notify(name, "invalid node probability given, not saved")
worldedit_commands/init.lua:		worldedit.player_notify(name, count .. " objects cleared")
worldedit_commands/cuboid.lua:			worldedit.player_notify(name, "Missing privileges: " ..
```

**Comment by alek13** [comment]
@sfan5 , I add missed translations. Also I found and add another bunch of missed in `init.lua`.
Also translate `safe.lua` & `cuboid.lua`.

But I left it as it is a couple of phrases in commands with lua-code execution, like:
```
worldedit.player_notify(name, "code error: " .. err)
worldedit.player_notify(name, "code successfully executed", false)
worldedit.player_notify(name, "code error: " .. err, false)
```
Because people who executes lua-code should know english, and anyway the `err` will be in english anyway

File: worldedit_commands/cuboid.lua [hunk 3]
```diff
@@ -28,18 +30,18 @@ worldedit.register_command("outset", {
 			assert(worldedit.cuboid_linear_expand(name, 'y', 1, amount))
 			assert(worldedit.cuboid_linear_expand(name, 'y', -1, amount))
 		else
-			return false, "Invalid number of arguments"
+			return false, S("Invalid number of arguments")
 		end
 
 		worldedit.marker_update(name)
-		return true, "Region outset by " .. amount .. " blocks"
+		return true, S("Region outset by @1 blocks", amount)
```
Comment (sfan5) [inline worldedit_commands/cuboid.lua:37]: sorry to ask for more work but if this is being touched anyway can you fix the terminology?
"blocks" is wrong and should be "nodes"

File: worldedit_commands/cuboid.lua [hunk 4]
```diff
@@ -202,13 +204,13 @@ worldedit.register_command("contract", {
...
 	end,
 })
 
 worldedit.register_command("cubeapply", {
 	params = "<size>/(<sizex> <sizey> <sizez>) <command> [parameters]",
-	description = "Select a cube with side length <size> around position 1 and run <command> on region",
+	description = S("Select a cube with side length <size> around position 1 and run <command> on region"),
 	privs = {worldedit=true},
 	require_pos = 1,
 	parse = function(param)
```
Comment (sfan5) [inline worldedit_commands/cuboid.lua:216]: below here you missed the "invalid usage:" string

File: worldedit_commands/cuboid.lua [hunk 5]
```diff
@@ -1,6 +1,8 @@
+local S = minetest.get_translator("worldedit")
```
[the remark this answers is not in the document]
Comment (alek13) [reply worldedit_commands/cuboid.lua:1]: its not necessary.. But, if translations will appears for `worldedit`, `..._gui`, ...
yep, I will fix

File: worldedit_commands/cuboid.lua (hunk 3 above)
Comment (alek13) [reply worldedit_commands/cuboid.lua]: yep, also fixed for `//inset` command

File: worldedit_commands/cuboid.lua (hunk 4 above)
Comment (alek13) [reply worldedit_commands/cuboid.lua:216]: thnx, fixed

File: worldedit_commands/locale/template.txt [hunk 6]
```diff
@@ -148,4 +148,8 @@ Region expanded by @1 nodes=
 Contracts the selection in the selected absolute or relative axis=
 Region contracted by @1 nodes=
 Select a cube with side length <size> around position 1 and run <command> on region=
+invalid usage: //@1 cannot be used with cubeapply=
 Missing privileges: @1=
+
+### wand.lua ###
+WorldEdit Wand tool@\nLeft-click to set 1st position, right-click to set 2nd=
```
Comment (sfan5) [inline worldedit_commands/locale/template.txt:155]: has to be `@n`

File: worldedit_commands/locale/worldedit.ru.tr [hunk 7]
```diff
@@ -1,4 +1,4 @@
-# textdomain: worldedit
+# textdomain: worldedit_commands
```
Comment (sfan5) [inline worldedit_commands/locale/worldedit.ru.tr:1]: doesn't this file also need to be renamed?
not sure about the convention

File: worldedit_commands/locale/template.txt (hunk 6 above)
Comment (alek13) [reply worldedit_commands/locale/template.txt:155]: https://minetest.gitlab.io/minetest/translations/#escapes
> - `@\n` (where the \n is a literal newline) acts as a literal newline. As with `@=`, this escape is not required in strings given to `minetest.translate`, but is in translation files.
> - `@n` acts as a literal newline as well.

You can use both. But!!, with `@n` not works as expected (maybe I made a mistake somewhere).
Anyway by documentation you can use both. And this works as expected.
Let's leave it as is.

File: worldedit_commands/locale/worldedit.ru.tr (hunk 7 above)
Comment (alek13) [reply worldedit_commands/locale/worldedit.ru.tr]: done

**Comment by sfan5** [comment]
Thanks for your contribution.

**Pull request merged**